pkg/apis/provisioning/v1alpha5/labels.go

```diff
@@ -43,6 +43,7 @@ var (
 	// they are not used in a context where they may be passed as argument to kubelet.
 	LabelDomainExceptions = sets.NewString(
 		"kops.k8s.io",
+		"node.kubernetes.io",
```

Comment:
I will hesitate to enable `v1.LabelNamespaceSuffixKubelet` yet because the domain is reserved for kubelet and we haven't done our due diligence to understand the impacts.